test/unit/lib/redmine/menu_manager/menu_helper_test.rb

```diff
@@ -84,7 +84,7 @@ def test_render_menu_node
 
     html_node = HTML::Document.new(@response.body)
     assert_select(html_node.root, "li") do
-      assert_select("a.single-node", "Single node")
+      assert_select("a.single-node-menu-item", "Single node")
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.